docs/kubernetes/mailu/webmail.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
       - name: roundcube
-        image: mailu/roundcube:1.5
+        image: mailu/roundcube:master
```

Comment:
I've seen this update to `master` in several places. I'm assuming this is just for development, but in a production environment, it can lead to problems if an untested change updates the image and the user's machine pulls it.

Reply:
As you can read above during this PR, it's just for testing purposes now. When the PR is finished @kaiyou will help decide if we step into a new version `1.6` (or whatever is decided), build new containers and then we change the container images versions in all of the manifests.